test/cpp/util/tls_test_utils.cc

```diff
@@ -43,6 +40,7 @@ AsyncCertificateVerifier::AsyncCertificateVerifier(bool success)
     : success_(success),
       thread_("AsyncCertificateVerifierWorkerThread", WorkerThread, this) {
   thread_.Start();
+  thread_started_ = true;
```

Comment:
Do we need this bool? Won't it always be true since it's hard-set during the ctor and not touched elsewhere?

Reply:
We have another ctor that doesn't set it (defined in the header since it only takes the status), and this bool defaults to false.

thread_.Join() will crash if it never started. That's why we need this boolean.